Sources/Brave/Extensions/UIAction+Extensions.swift

```diff
@@ -0,0 +1,54 @@
+// Copyright 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import UIKit
+import Strings
+import BraveCore
+
+extension UIAction {
+  static func makePasteAndGoAction(pasteCallback: @escaping (String) -> Void) -> UIAction {
```

Comment:
None of these really belong in a generic extension on `UIAction`

Reply:
I just want to share the code between those two components. Is there any suggestion you may have?

Reply:
A `UrlUIActions` struct maybe? This way you do not expose those actions for every action

Reply:
what if we just prefix brave on this. I mean we do this to colors and other things so we have `Color.bravePrimary` and I think now we will have `Image.braveBroom` (I'm not sure if this is what direction we decided). What's the problem with having `UIAction.bravePasteAndGo(pasteCallback:)`